Cargo.toml

```diff
@@ -1,25 +1,20 @@
 [package]
+version = "0.9.0"
```

Comment:
Are you skipping 0.8.0 on purpose? 😅

Reply:
Yeah, I thought it should match the 52-series. I thought semver allowed skipping, but I'm more than happy to change it back if you'd like :)

Reply:
Nah, it's fine. Just curious...